source/codegen/metadata/nidcpower/functions.py

```diff
@@ -3585,6 +3752,7 @@
         'returns': 'ViStatus'
     },
     'WaitForEventWithChannels': {
+        'codegen_method': 'public',
```

Comment:
I understand why we specify the codegen method in hapigen. But why does its output include that in the metadata here? It certainly wasn't needed before.

Reply:
We talked offline but we'll leave this in the hapigen generation and output metadata files for the MI drivers. Causes more noise in the diffs right now but fits the vision of being explicit in the hapigen generation and resulting output metadata.

Main reason is that we want to be explicit where we can in the hapigen plugin to specify defaults to ideally have the output metadata fed in to grpc_device without any further manipulation of the metadata by the grpc_device process. And generally, I like the direction of being more explicit in the source grpc_device metadata when possible so we don't have to know about what grpc_device will default missing keys/values to.